cmd/trigger_ui/main.go

```diff
@@ -279,21 +421,52 @@ func main() {
 
 	router.LoadHTMLGlob("./templates/*")
 
+	sqlite_path, exists := os.LookupEnv("TRIGGER_DB")
+	if !exists {
+		panic("No TRIGGER_DB environment variable.")
+	}
+
+	db, err := sql.Open("sqlite3", sqlite_path)
+	if err != nil {
+		panic(err.Error())
+	}
+
+	_, err = db.Exec(`CREATE TABLE IF NOT EXISTS tags (
+		name TEXT NOT NULL
+	);
+	CREATE TABLE IF NOT EXISTS persons (
+		name TEXT NOT NULL
+	);
+	CREATE TABLE IF NOT EXISTS programID (
+		name TEXT NOT NULL
+	);
+	`)
+	if err != nil {
+		panic(err.Error())
+	}
+
 	server := &TriggerServer{
 		vsapiClient,
 		assetExportDestinations,
 		wfClient,
 		lang,
 		ExportAudioSources,
+		db,
 	}
 
 	router.GET("/vx-export", server.triggerHandlerGET)
 	router.GET("/vx-export/list", server.listGET)
+	router.GET("/upload-master", server.uploadMasterGET)
```

Comment:
Instead of creating a lot of duplicate routes, you can create a router group here.

group := router.Group("/upload-master")
group.GET("/", ...)
group.POST("/", ...)